tests/test_client_mock.py

```diff
@@ -1,24 +1,25 @@
 import pytest
 
-from eth_utils import decode_hex, is_same_address
+from eth_utils import decode_hex, is_same_address, is_bytes
 
 from raiden_libs.utils.signing import eth_verify
 
 
 def test_client_multiple_topups(generate_raiden_clients):
     deposits = [1, 1, 2, 3, 5, 8, 13]
     c1, c2 = generate_raiden_clients(2)
-    channel_id = c1.open_channel(c2.address)
-    assert channel_id > 0
+    channel_identifier = c1.open_channel(c2.address)
+    assert is_bytes(channel_identifier)
```

Comment:
this should be `is_instance(channel_identifier, str)` no? Or even better a custom function 

```
def is_channel_identifier(channel_id):
   return len(decode_hex(channel_id)) == 32
```

Reply:
Fixed